0-bootstrap/main.tf

```diff
@@ -101,8 +101,7 @@ resource "google_billing_account_iam_member" "tf_billing_admin" {
 
 // Comment-out the cloudbuild_bootstrap module and its outputs if you want to use Jenkins instead of Cloud Build
 module "cloudbuild_bootstrap" {
-  source                      = "terraform-google-modules/bootstrap/google//modules/cloudbuild"
-  version                     = "~> 5.0"
+  source                      = "github.com/terraform-google-modules/terraform-google-bootstrap//modules/cloudbuild"
```

Comment:
Hi @iyabchen to be able to use the latest version of the `cloudbuild` module it is also necessary to use the latest version of the main bootstrap module for the seed module 
https://github.com/terraform-google-modules/terraform-example-foundation/blob/d940f6e3401ad3f19987acf56ba70b086bb2a855/0-bootstrap/main.tf#L32-L34

because the cloud build module depends on `enable_cross_project_service_account_usage`
https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/a300b9c27796200f2a565173484eb997d8e296a8/main.tf#L72

```hcl
module "enable_cross_project_service_account_usage" {
  source  = "terraform-google-modules/org-policy/google"
  version = "~> 5.1"

  project_id  = module.seed_project.project_id
  policy_for  = "project"
  policy_type = "boolean"
  enforce     = "false"
  constraint  = "constraints/iam.disableCrossProjectServiceAccountUsage"
}
```